conf/application.conf

```diff
@@ -103,6 +105,10 @@ controllers {
 
 # Microservice specific config
 
```

Comment:
do we need lock collection to be in `customs-notification` or `customs-notification-locks` database? Maybe one for Usman

Reply:
Think we need to consider how we do this once we get the two repo stories played and see who they sit together as we are relying on 3rd party library.